src/test/java/com/example/hello/HelloApplicationTests.java

```diff
@@ -6,8 +6,9 @@
 @SpringBootTest
 class HelloApplicationTests {
 
-	@Test
```

Comment:
there was no significant change here, why the edit?

Reply:
The test prevented me from installing it

Reply:
No, it can't prevent you from adding  dependencies